src/main/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -368,11 +366,18 @@ const showLine = (config, boxPath) =>
             d.high &&
             d.low &&
             utils.hasValue(config.shownTargets, d.high.key) &&
-            utils.hasValue(config.shownTargets, d.low.key);
+            utils.hasValue(config.shownTargets, d.low.key) &&
+            (document.querySelector(`.${styles.legendItem}[aria-describedby="${d.high.key}"]`) ?.getAttribute("aria-current") === "true") 
+            &&
+            (document.querySelector(`.${styles.legendItem}[aria-describedby="${d.low.key}"]`) ?.getAttribute("aria-current") === "true");
         return d3
             .select(this)
             .select(`.${styles.pairedLine}`)
-            .attr("aria-hidden", (d) => !shouldCreateLine(d));
+            .attr("aria-hidden", (d) => {
+                console.log(shouldCreateLine(d));
```

Comment:
nit: can we remove the console.log statement here?

Reply:
Removed it here https://github.com/cerner/carbon-graphs/pull/138/commits/2da68b4daa7cd2ee30db5f7473727625d7347e55.